Geneious/Geneious.download.recipe

```diff
@@ -35,6 +35,19 @@ See http://desktop-links.geneious.com/assets/installers/geneious/GeneiousVersion
 				<string>%MAJOR_VERSION% %RELEASE% %RELEASE%.*(Geneious.*mac.*dmg)</string>
```

Comment:
Instead of using a new processor, just use named capture groups and change the dash later:
```2019 release release (?P<PRODUCT>[^-]+)-(?P<DISTRIBUTION>.*\.dmg)```
note that you'll need to escape the brackets to `&lt;` and `&gt;` because XML.

Reply:
For URL downloader, you'll change the download URL to
`<string>%DOWNLOAD_BASE_URL%/%RELEASE%/%PRODUCT%_%DISTRIBUTION%</string>`

Reply:
Please also update the input MAJOR_VERSION to be 2019 instead of 10.

Reply:
Hey @cwhits ,
thanks for your suggestions, I just added them to my testing environment. Unfortunately, your change would break backwards compatibility for Geneious versions 10 and 11 which we still want to provide. By using the replace processor this is not a problem and existing recipes can still use the version input key for versions older than 2019.

Reply:
Updated `MAJOR_VERSION` to 2019

Reply:
will there find and replace processor work for this? If not, put the regex processor in shared processors, please.

Reply:
It will for sure work for this case but could potentially be used by other recipes too. I did not see the `SharedProcessors` folder before. I will add the processor there.